content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -0,0 +1,90 @@
+---
+layout: developersection
+title: Dependency Management
+references:
+- url: https://github.com/jenkinsci/plugin-pom/
+  title: Plugin parent POM 4.0 and up
+- url: https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml
+  title: Jenkins Core BOM
+---
+
+This section details some ways to simplify the management of dependency versions.
+
+== Jenkins Core BOM
+
+Since version 2.195, Jenkins Core provides a  link:https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[Maven Bill Of Materials (BOM)]
```

Comment:
For further enhancement purposes, it would be great to have one sentence per line. @daniel-beck said it is the current recommendation for these docs are (see https://github.com/jenkins-infra/jenkins.io/pull/2544#discussion_r331779918 )